File: Leetcode/3021 - Alice and Bob Playing Flower Game/solution.cpp
long long flowerGame(int n, int m) {
    long long nEven = n / 2;
    long long nOdd = (n + 1) / 2;
    long long mEven = m / 2;
    long long mOdd = (m + 1) / 2;
    return (nEven * mOdd) + (nOdd * mEven);
}

File: Leetcode/3021 - Alice and Bob Playing Flower Game/solution.py
def flowerGame(n: int, m: int) -> int:
    n_even: int = n // 2
    m_even: int = m // 2
    n_odd: int = (n + 1) // 2
    m_odd: int = (m + 1) // 2
    return (n_even * m_odd) + (n_odd * m_even)